src/test/kotlin/org/opensearch/replication/integ/rest/UpdateAutoFollowPatternIT.kt

```diff
@@ -95,6 +95,7 @@ class UpdateAutoFollowPatternIT: MultiClusterRestTestCase() {
                 var af_stats = stats.get("autofollow_stats")!! as ArrayList<HashMap<String, Any>>
                 for (key in af_stats) {
                     assert(key["num_success_start_replication"]!! as Int == 2)
+                    assert(key["last_execution_time"]!! as Long != 0L)
```

Comment:
Could we add checks after the poll duration as well?

Reply:
Added.